WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -194,10 +218,11 @@ private void showRoleChangeDialog() {
     }
 
     // used to optimistically update the role
-    public void changeRole(Role newRole) {
-        mRoleTextView.setText(newRole.toDisplayString());
+    public void changeRole(String newRole) {
+        mRoleTextView.setText(RoleUtils.getDisplayName(newRole, mUserRoles));
     }
 
+    @SuppressLint("ObsoleteSdkInt")
```

Comment:
Rather than adding this suppression could we simply remove the version check? Our minimum supported API is 16 so it should never fail.